### ISSUE TITLE
When and how did you start learning about web development?

### ISSUE THREAD

**Issue by silvenon** [arthurvr/ama#13]
I cleverly disguised two questions into one.

**Comment by arthurvr** [comment]
I was about 7 or 8 years old and got into writing some basic HTML. I don't know why I wanted to build websites back then, but I did :stuck_out_tongue_winking_eye:. I guess I just wanted to build collections of gifs and pictures for lolz.

And that pretty much summarises what I did with the web the first years I was into it.

Little later I picked up some Ruby, CSS and JavaScript books, read them, and never stopped learning more. I loved coding and never stopped doing it. Wrestling through the English in the books was a harder challenge than the coding I should say.

Sorry for the short answer but honestly, it's hard to say. I just fell in love with coding and never stopped doing it. Thanks for the question!

*If that expresses how nerdy I am: I wrote vim script to solve my homework back when I was 12.*
